### PR TITLE
Update expected values in SGX attestation tests

### LINKED ISSUE #302: Fix SGX bug with get_att unit tests
They were failing.

### PR DESCRIPTION
**Pull request by lkatalin**
This allows the `req_ti` and `req_quote` unit tests to pass again. They became outdated when we moved to the V3 Quote.

Resolves #302

### CODE REVIEW

File: tests/bin/sgx_get_att_quote.c [hunk 1]
```diff
@@ -22,5 +34,14 @@ int main(void) {
     if (size < 0)
         return 1;
 
+    /* check beginning of quote matches expected value */
+    for (i; i < 32; i++)
+    {
+        if (buf[i] != expected[i] && (buf[i])!= dummy[i])
```
Comment (haraldh) [inline tests/bin/sgx_get_att_quote.c:40]: odd  parentheses

**Review by haraldh — DISMISSED** [review]

File: tests/bin/sgx_get_att_quote.c (hunk 1 above)
Comment (haraldh) [reply tests/bin/sgx_get_att_quote.c:40]: lol.. I wondered why `cargo clippy` didn't complain :)

**Review by haraldh — APPROVED** [review]

File: tests/bin/sgx_get_att_quote.c (hunk 1 above)
Comment (lkatalin) [reply tests/bin/sgx_get_att_quote.c]: You're right, that is weird... why did I write that? I fixed it. : )

Comment (haraldh) [reply tests/bin/sgx_get_att_quote.c]: >> lol.. I wondered why cargo clippy didn't complain :)

> You're right, that is weird...

Not sure, if you recognized, that this is a C-file and not rust :)

**Review by mbestavros — APPROVED** [review]

**Review by connorkuehl — APPROVED** [review]

**Pull request merged**